stdlib/TOML/src/print.jl

```diff
@@ -93,7 +93,8 @@ function printvalue(f::MbyFunc, io::IO, value::TOMLValue)
     value isa Dates.Time     ? Base.print(io, Dates.format(value, Dates.dateformat"HH:MM:SS.sss")) :
     value isa Dates.Date     ? Base.print(io, Dates.format(value, Dates.dateformat"YYYY-mm-dd")) :
     value isa Bool           ? Base.print(io, value ? "true" : "false") :
-    value isa Integer        ? Base.print(io, Int64(value)) :  # TOML specifies 64-bit signed long range for integer
+    value isa Signed         ? Base.show(io, value) :  # TOML specifies 64-bit signed long range for integer
+    value isa Unsigned       ? Base.show(io, value) :  # TOML specifies 64-bit unsigned long range for integer
```

Comment:
What does "TOML specifies" mean here?

Reply:
oh...lol it's copy-pasted from the previous 

https://github.com/JuliaLang/julia/pull/47903/files#diff-30a7c615c5a56e6bec2fc0761e54da476f581123fa40d40b5aea56b082132015L96

```julia
value isa Integer ? Base.print(io, Int64(value)) : # TOML specifies 64-bit signed long range for integer
```

I'm not sure what it means actually maybe we should just remove these comments since we are not converting them to `Int64` anymore but use the default Julia printing. The default Julia printing seems to be the same with the syntax of TOML, or should we only convert Signed `<64` to `Int64`? I think both are compatible with what was supported previously

Reply:
The earlier comment said that TOML values are Int64 so that's why we explicitly convert to `Int64`. I think it is fine to remove the comment, but why not just have `value isa Integer ? Base.show(io, value)`?

Reply:
> why not just have value isa Integer ? Base.show(io, value)?

Good point, I don't remember, maybe cuz I was drunk 🤷🏼‍♂️